pkg/volume/plugins.go

```diff
@@ -42,6 +42,13 @@ type VolumeOptions struct {
 	RootContext string
 }
 
+// System is an interface that abstracts certain operations on a host operating
+// system that a volume might want to execute.
+type System interface {
```

Comment:
I think you misspelled Host.GetMounter() and Host.WriteFile()

In all seriousness, the VolumeHost is provided to plugins as THE way to interface with the app that hosts the plugin.  I do not think we should be injecting more objects from the host's plane into the plugin's plane.